nixos/tests/soju.nix

```diff
@@ -0,0 +1,36 @@
+import ./make-test-python.nix ({ pkgs, lib, ... }:
+let
+  domain = "soju.localdomain";
+  user = "testuser";
+  pass = "hunter2";
+
+  tls-cert = pkgs.runCommand "selfSignedCerts" { buildInputs = [ pkgs.openssl ]; } ''
+    openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -nodes -days 36500 \
+      -subj '/CN=${domain}' -extensions v3_req \
+      -addext 'subjectAltName = DNS:*.${domain}'
+    install -D -t $out key.pem cert.pem
+  '';
```

Comment:
Can't we use the existing snake oil certs?

Reply:
Didn't know about these thanks!